pgd/settings.py

```diff
@@ -28,7 +28,7 @@
 }
 import sys
 if 'test' in sys.argv:
-    DATABASES['default']['OPTIONS'] = {'init_command': 'SET foreign_key_checks=0'}
+    DATABASE_OPTIONS = {'init_command': 'SET foreign_key_checks=0'}
```

Comment:
Why did you make this change?  If it's because of the comments in `pgd_splicer/tests.py`, those comments are pre-Django-1.6 and should be removed since the existing way is the right way in Django 1.6.